db/migrate/20201203154533_add_criteria_to_placement_needs.rb

```diff
@@ -0,0 +1,5 @@
+class AddCriteriaToPlacementNeeds < ActiveRecord::Migration[6.0]
+  def change
+    add_column :placement_needs, :criteria, :string
```

Comment:
If we are replacing the checkboxes, than I'd remove them so that we don't keep piling problems into the data.

Reply:
rm  booleans. not sure what to call criteria, keep it for now

Reply:
I think it's `placement_type` best to describe the content. But we can rename it later.